### PR TITLE
MC-12120: create storage class doc added

### PR DESCRIPTION
**Pull request by AnKhangC**
#### Changes made
Screen Shot 2020-08-03 at 2 26 58 PM

#### Related PRs
- https://github.com/cloudops/cloudmc-kubernetes-sdk/pull/136

### CODE REVIEW

**Review by patrickspensieri — COMMENTED** [review]
Looks like we're just missing the doc for the Kubernetes extensions like GCP/Azure:)
Should be very similar to this, except
- extra level of indentation (extra `#` on section headers)
- requires the `cluster_id` query param

File: source/includes/kubernetes/_k8_storageclasses.md [hunk 1]
```diff
@@ -130,6 +130,60 @@ Retrieve a storage class and all its info in a given [environment](#administrati
...
+
+Create a storage class in a given [environment](#administration-environments).
+
+| Required Attributes                        | &nbsp;                                                                      |
+| ------------------------------------------ | ----------------------------------------------------------------------------|
+| `apiVersion` <br/> _string_                | The api version (versioned schema) of the storage class.                    |
+| `kind`<br/>_string_                        | The string value representing the REST resource this object represents.     |
+| `metadata` <br/>_object_                   | The metadata of the storage class.                                          |
+| `metadata.name` <br/>_string_              | The name of the storage class.                                              |
+| `provisioner` <br/>_string_                | The provsioner for the storage class                                        |
```
Comment (patrickspensieri) [inline source/includes/kubernetes/_k8_storageclasses.md]: Should be _provisioner_

**Review by patrickspensieri — APPROVED** [review]
Looks good! Just a small typo in both docs

File: source/includes/kubernetes/_k8_storageclasses.md (hunk 1 above)
Comment (AnKhangC) [reply source/includes/kubernetes/_k8_storageclasses.md]: 77b51ad

**Review by evanluc — APPROVED** [review]

**Pull request merged**